src/scrapers/union-bank.ts

```diff
@@ -274,10 +274,7 @@ async function fetchAccounts(page: Page, startDate: Moment) {
 }
 
 async function waitForPostLogin(page: Page) {
-  return Promise.race([
-    waitUntilElementFound(page, '#signoff', true),
-    waitUntilElementFound(page, '#restore', true),
```

Comment:
why did you remove the #restore part? this is basically the view that is shown once in a while to replace a password? I don't have union so I cannot verify it but based on other scrapers usually here the scraper wait for either a valid login or a password expired status hence the promise.race

consider keeping the restore part